Calibration/HcalAlCaRecoProducers/plugins/AlCaIsoTracksFilter.cc

```diff
@@ -84,19 +84,21 @@ class AlCaIsoTracksFilter : public edm::stream::EDFilter<edm::GlobalCache<AlCaIs
   void endRun(edm::Run const&, edm::EventSetup const&) override;
   
   // ----------member data ---------------------------
-  HLTConfigProvider             hltConfig_;
-  std::vector<std::string>      trigNames_, HLTNames_;
-  spr::trackSelectionParameters selectionParameter_;
-  std::string                   theTrackQuality_, processName_;
-  double                        maxRestrictionP_, slopeRestrictionP_;
-  double                        a_mipR_, a_coneR_, a_charIsoR_;
-  double                        pTrackMin_, eEcalMax_, eIsolate_;
-  double                        pTrackLow_, pTrackHigh_;
-  int                           preScale_;
-  unsigned int                  nRun_, nAll_, nGood_, nRange_;
-  edm::InputTag                 triggerEvent_, theTriggerResultsLabel;
-  edm::InputTag                 labelGenTrack_, labelRecVtx_;
-  edm::InputTag                 labelEB_, labelEE_, labelHBHE_;
+  HLTConfigProvider              hltConfig_;
+  std::vector<std::string>       HLTNames_;
```

Comment:
I see that you do not make this one const, while you changed `trigNames_` to a const member.
But now I am failing to see where `HLTNames_` is used at all. Could this maybe just go?